### DISCUSSION TITLE
Streamlink Twitch GUI blank screen and errors

### DISCUSSION THREAD

**Discussion by tangosox** [streamlink/streamlink-twitch-gui#1019] [Q&A] [answered]
I've been running the app image of streamlink gui with no problem up until today.  It's been a few days since I've used it and I had a couple updates on my OS (openSUSE Tumbleweed, kernel 6.10.2-1) and I'm not sure if that's the issue.  I also compiled it from source and have the same exact problem.  It's just a blank grey screen and terminal is scrolling by too fast and far to see what the many errors are or if they are relevant.  Only one I can see is at the end that says "Errors:Location of variable sk_FragColor conflicts with another variable."  After quitting the app I get "[11730:11738:0806/113554.438782:ERROR:command_buffer_proxy_impl.cc(319)] GPU state invalid after WaitForGetOffsetInRange."
Any idea what's happening or if theres a way to get better logs/error messages? Was really enjoying the app and don't understand why it stopped working.

**Comment by bastimeyer** [comment, marked as answer]
https://github.com/streamlink/streamlink-twitch-gui/issues/1017#issue-2309172972

```sh
rm -r "${XDG_CONFIG_HOME:-~/.config}"/streamlink-twitch-gui/Default/GPUCache
```

**Reply by tangosox** [reply]
Thank you! Any idea why that happened?

**Reply by bastimeyer** [reply]
It's right there in the linked thread. NW.js doesn't invalidate its shader cache when a new LLVM version is used, which may break previously compiled shaders. It only does that when its own version is bumped. This is a bug in NW.js/Chromium which needs to be fixed there, but as you can see in #1015, I can't bump NW.js easily should there be a fix in recent versions.

Btw, the discussion forum is not a bug tracker, so I'm closing this now. It's also not "Streamlink" or "Streamlink GUI", it's Streamlink Twitch GUI.
